Jenkinsfile

```diff
@@ -125,7 +125,7 @@ pipeline {
                     -e DOCKER_EXPERIMENTAL=y \
                     --name docker-pr-exp$BUILD_NUMBER \
                     -e DOCKER_GITCOMMIT=${GITCOMMIT} \
-                    -e DOCKER_GRAPHDRIVER=vfs \
+                    -e DOCKER_GRAPHDRIVER=overlay2 \
```

Comment:
Would it work if we _removed_ `=overlay2` ? Then we can set `DOCKER_GRAPHDRIVER` in jenkins-configuration if we want to change it (thinking out loud)

Reply:
for now, trying to keep number of knobs to turn to a minimum. perhaps in the future, but wants to get the jenkinsfile working and dependable first.